components/brave_wallet_ui/components/desktop/popup-modals/edit-visible-assets-modal/style.ts

```diff
@@ -57,44 +42,82 @@ export const Divider = styled.div`
   background-color: ${(p) => p.theme.color.divider01};
 `
 
-export const NoAssetRow = styled.div`
-  display: flex;
-  flex-direction: row;
-  align-items: flex-start;
-  justify-content: flex-start;
-  width: 100%;
-  margin-top: 15px;
-`
-
-export const NoAssetText = styled.span`
-  font-family: Poppins;
-  font-size: 14px;
-  letter-spacing: 0.01em;
-  line-height: 20px;
-  color: ${(p) => p.theme.color.text02};
-`
-
 export const NoAssetButton = styled(WalletButton)`
   cursor: pointer;
   outline: none;
   border: none;
   background: none;
   padding: 0px;
-  margin: 10px 0px;
+  margin: 0px;
   font-family: Poppins;
   font-size: 14px;
-  letter-spacing: 0.01em;
+  letter-spacing: 0.4px;
   line-height: 20px;
-  color: ${(p) => p.theme.palette.blurple500};
-  @media (prefers-color-scheme: dark) {
-    color: ${(p) => p.theme.color.interactive06};
+  font-weight: 600;
+  color: ${leo.color.text.interactive};
+`
+
+export const ResponsiveRow = styled(Row)`
+  padding: 0px 32px;
+  @media screen and (max-width: ${layoutPanelWidth}px) {
+    padding: 0px 16px;
+  }
+`
```

Comment:
Mayb let's declare this to the popupmodals folder since it is shared between 2 components that are modals?
Can we also rename this to somethig more specific? Maybe something like `PaddedControlsRow`?

Reply:
Moved!